raiden/tests/unit/transfer/test_node.py

```diff
@@ -56,3 +114,331 @@ def test_is_transaction_effect_satisfied(
+    assert not transition_result.events
+
+    mediator_state = MediatorTransferState(
+        secrethash=UNIT_SECRETHASH,
+        routes=[
+            RouteState(
+                route=[netting_channel_state.partner_state.address],
+                forward_channel_id=netting_channel_state.canonical_identifier.channel_identifier,
+            )
+        ],
+    )
+    subtask = MediatorTask(
+        token_network_address=netting_channel_state.canonical_identifier.token_network_address,
+        mediator_state=mediator_state,
+    )
+    chain_state.payment_mapping.secrethashes_to_task[UNIT_SECRETHASH] = subtask
+
+    lock = factories.HashTimeLockState(amount=0, expiration=2, secrethash=UNIT_SECRETHASH)
+
+    netting_channel_state.partner_state.secrethashes_to_lockedlocks[UNIT_SECRETHASH] = lock
+    netting_channel_state.partner_state.merkletree = compute_merkletree_with(
+        merkletree=make_empty_merkle_tree(), lockhash=lock.lockhash
+    )
+    result = object()
+    monkeypatch.setattr(
+        raiden.transfer.node,
+        "subdispatch_mediatortask",
+        lambda *args, **kwargs: TransitionResult(chain_state, [result]),
+    )
+    transition_result = handle_node_change_network_state(chain_state, state_change)
+
+    assert transition_result.events == [result]
+
+
+def test_handle_new_payment_network(chain_state, token_network_address):
+    token_address = factories.make_address()
+    token_network = TokenNetworkState(
+        address=token_network_address,
+        token_address=token_address,
+        network_graph=TokenNetworkGraphState(token_network_address=token_network_address),
+    )
+    payment_network = PaymentNetworkState(
+        address=factories.make_address(), token_network_list=[token_network]
+    )
+    state_change = ContractReceiveNewPaymentNetwork(
+        transaction_hash=factories.make_transaction_hash(),
+        payment_network=payment_network,
+        block_hash=make_block_hash(),
+        block_number=1,
+    )
+    assert payment_network.address not in chain_state.identifiers_to_paymentnetworks
+    transition_result = handle_new_payment_network(chain_state, state_change)
+    assert transition_result.new_state == chain_state
+    msg = "handle_new_payment_network did not add to chain_state mapping"
+    assert payment_network.address in chain_state.identifiers_to_paymentnetworks, msg
+
+
+def test_inplace_delete_message_queue(chain_state):
+    sender = factories.make_address()
+    canonical_identifier = factories.make_canonical_identifier()
+    message_id = factories.make_message_identifier()
+    delivered_state_change = ReceiveDelivered(sender=sender, message_identifier=message_id)
+    handle_delivered(chain_state=chain_state, state_change=delivered_state_change)
+
+    processed_state_change = ReceiveProcessed(sender=sender, message_identifier=message_id)
+    handle_processed(chain_state=chain_state, state_change=processed_state_change)
```

Comment:
What's the point of handling delivered and processed here?

Reply:
unnecessary, removed. thanks!